### PR TITLE
Fix MediaTypeOptions methods typo

### LINKED ISSUE #22431: Http Logging Middleware - MediaTypeOptions doc typos
HttpLogging Middleware documentation has typos in MediaTypeOptions section.

image

Just logging an issue here so that i can create a PR fix.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 7c772b2a-5d37-84af-624c-9022efe0758b
* Version Independent ID: b39594cc-9d1c-611c-e3e7-8755d74253a6
* Content: [HTTP Logging in .NET Core and ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/fundamentals/http-logging/?view=aspnetcore-6.0)
* Content Source: [aspnetcore/fundamentals/http-logging/index.md](https://github.com/dotnet/AspNetCore.Docs/blob/main/aspnetcore/fundamentals/http-logging/index.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-fundamentals**
* GitHub Login: @jkotalik
* Microsoft Alias: **jukotali**

### PR DESCRIPTION
**Pull request by lohithgn**
Fixes #22431

Fix AddText() and AddBinary() methods documentation

### CODE REVIEW

File: aspnetcore/fundamentals/http-logging/index.md [hunk 1]
```diff
@@ -97,13 +97,13 @@ To configure the HTTP logging middleware, call `AddHttpLogging` in `ConfigureSer
 
 <!-- When API is published, replace the following with API links -->
 
-* `public void AddText(string contentType, Encoding encoding)`
+* `public void AddText(string contentType)`
```
Comment (Rick-Anderson) [inline aspnetcore/fundamentals/http-logging/index.md:100]: revert

Comment (Rick-Anderson) [inline aspnetcore/fundamentals/http-logging/index.md:100]:
```suggestion
`public void AddText(string contentType, Encoding encoding)`
```

**Review by Rick-Anderson — APPROVED** [review]

**Comment by Rick-Anderson** [comment]
@lohithgn  thanks for the fix. Sorry I missed the other overload.

**Comment by lohithgn** [comment]
@Rick-Anderson  Thanks. Appreciate the quick responses :)

**Pull request merged**